wizard/protected/models/Policy.php

```
@@ -132,7 +132,7 @@ public function hasPolicy($external_id){
 		
 	}
 	public function isPublished($external_id){
-		if(Policy::model()->findByAttributes(array('EXTERNAL_ID'=>$external_id))->IS_PUBLISHED == 1)
+		if(@Policy::model()->findByAttributes(array('EXTERNAL_ID'=>$external_id))->IS_PUBLISHED == 1)
```

Wouldn't it be better to use [exists()](http://www.yiiframework.com/doc/api/1.1/CActiveRecord#exists-detail) instead of suppressing the error when no record is found?